pyvo/mivot/features/static_reference_resolver.py

```diff
@@ -0,0 +1,103 @@
+"""
+Created on 22 Dec 2021
+
+@author: laurentmichel
+"""
+from copy import deepcopy
+from pyvo.mivot.utils.exceptions import ResolveException
+from pyvo.mivot.utils.xml_utils import XmlUtils
+from pyvo.utils.prototype import prototype_feature
+
+
+@prototype_feature('MIVOT')
+class StaticReferenceResolver(object):
```

Comment:
drop `(object)` from class definitions. Here and everywhere below, too

```suggestion
class StaticReferenceResolver:
```